client/contract_example/cs/src/ExampleContract.cs

```diff
@@ -72,7 +72,7 @@ public ObjectHandle ObjectCreate(Baseline baseline, byte u8_0, byte u8_1, sbyte
                 throw new OwnershipException("`str_0` must be owned");
             }
             var p = new Ptr<ObjectHandle>(generated.TpContractExampleExampleContractObjectCreate(this.Inner.Value.p, baseline.Inner.Value.p, u8_0, u8_1, i8_0, i8_1, f32_0, f32_1, str_0.StealInner().p));
-            return new ObjectHandle(p);
+            return new ObjectHandle(p, OwnershipSemantics.SharedRef);
```

Comment:
Should be `Owned`: https://github.com/0xTELEPORTAL/constellation/blob/2f90dae624ebb56d001bc0f52674a2311c89149a/client/contract_example/rust/src/lib.rs#L78-L88